profile.md

```diff
@@ -508,6 +508,12 @@ in the `metadata` objects, it is RECOMMENDED that the following names
 be used when possible in an attempt to provide some degree of interoperability
 and consistency.
 
+Please note that in addition service metadata returned from [catalog endpoint](spec.md#catalog-management), 
```

Comment:
Shouldn't this be "Please note that in addition to service metadata..."?

Reply:
Good catch, thanks! Fixed in https://github.com/openservicebrokerapi/servicebroker/pull/730/commits/6a96926e5ebd28f21865aa5c7bff601454c09f32